pkg/util/stats_tracker.go

```diff
@@ -161,3 +162,21 @@ func (s *StatsTracker) dropOldPoints(now int64) {
 		s.peakPointsHead = nil
 	}
 }
+
+func (s *StatsTracker) InfoKey() string {
+	return "Latency Stats"
```

Comment:
```suggestion
	return "Pipeline Latency"
```

I think this is slightly more descriptive of what the metrics mean.